forms/modal.css

```diff
@@ -0,0 +1,85 @@
+/* Modal */
+/* The Modal (background) */
+.modal {
```

Comment:
Dit is eigelijk niet de modal zelf, maar de "modal page blocker" (zelf verzonnen woord).
Wat jij de modal-content noemt is eigelijk de echte "modal".

Binnen deze "echte" modal komt dan weer de content van de modal.
Doel is hier dat de content binnen de model niet gekoppeld is met de modal